src/lib/route.js

```diff
@@ -31,7 +31,7 @@ class Route {
     // if targetPrefix is specified, then destinations matching 'targetPrefix'
     // will follow this route, rather than destinations matching
     // 'destinationLedger'
-    this.targetPrefix = info.targetPrefix || this.destinationLedger
+    this.targetPrefix = info.targetPrefix == null ? this.destinationLedger : info.targetPrefix
```

Comment:
Why keep this distinction? In which situation would you specify a targetPrefix that's different from the destinationLedger?